browser/src/slideshow/engine/SlideShowHandler.ts

```diff
@@ -900,13 +900,49 @@ class SlideShowHandler {
 		return this.theMetaPres.getSlideInfoByIndex(nSlideIndex);
 	}
 
-	private getTexture(nSlideIndex: number): WebGLTexture | ImageBitmap | null {
+	private isMipMapsEnable(transitionFilterInfo: TransitionFilterInfo): boolean {
+		const allowedSubtypes = new Set([
```

Comment:
Maybe it would be better to move the set as a global structure `TransitionsWithMipMapEnabled` or any name that makes sense. So we don't need to create it on each `isMipMapEnabled` invocation

Reply:
make sense, thanks :)